examples/all-clusters-app/all-clusters-common/src/tls-certificate-management-instance.cpp

```diff
@@ -252,7 +253,7 @@ CHIP_ERROR TlsCertificateManagementCommandDelegate::LookupRootCert(EndpointId ma
                                                                    const ByteSpan & certificate,
                                                                    LoadedRootCertificateCallback loadedCallback) const
 {
-    std::array<uint8_t, chip::Crypto::kSHA1_Hash_Length> fingerprintPayload = { 0 };
+    std::array<uint8_t, Crypto::kSHA1_Hash_Length> fingerprintPayload = { 0 };
     MutableByteSpan calculatedFingerprint(fingerprintPayload);
     ReturnErrorOnFailure(Hash_SHA1(certificate.data(), certificate.size(), fingerprintPayload.data()));
```

Comment:
Needs to be SHA256 here too right?

Reply:
Thanks! Strange that this wasn't caught, though.... you'd think one of the test cases would fail. Looking into it

Reply:
Looping back, it was failing in the last test run 🤣 2.2 step 7, so we're good

But strangely all tests were passing in my "TC Tests Completed" commit, so now I'm just confused!